specification/applicationinsights/data-plane/LiveMetrics/preview/2024-04-01-preview/livemetrics.json

```diff
@@ -0,0 +1,1213 @@
+          },
+          {
+            "name": "Event",
+            "value": "Event",
+            "description": "Represents an event telemetry type."
+          },
+          {
+            "name": "Metric",
+            "value": "Metric",
+            "description": "Represents a metric telemetry type."
+          },
+          {
+            "name": "PerformanceCounter",
+            "value": "PerformanceCounter",
+            "description": "Represents a performance counter telemetry type."
+          },
+          {
+            "name": "Trace",
+            "value": "Trace",
+            "description": "Represents a trace telemetry type."
+          }
+        ]
+      }
+    },
+    "Trace": {
+      "type": "object",
+      "description": "Trace document type.",
+      "properties": {
+        "Message": {
+          "type": "string",
+          "description": "Trace message.",
+          "maxLength": 32768
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DocumentIngress"
+        }
+      ],
+      "x-ms-discriminator-value": "Trace"
+    },
+    "Versions": {
+      "type": "string",
+      "description": "Live Metrics service versions.",
+      "enum": [
+        "2024-04-01-preview"
+      ],
+      "x-ms-enum": {
+        "name": "Versions",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "v2024_04_01_preview",
+            "value": "2024-04-01-preview",
+            "description": "The 2024-04-01-preview version of the Live Metrics service."
+          }
+        ]
+      }
+    }
+  },
+  "parameters": {
+    "Azure.Core.Foundations.ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to use for this operation.",
+      "required": true,
+      "type": "string",
+      "minLength": 1,
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "apiVersion"
+    },
+    "IsSubscribedOperationRequestParameters.apiKey": {
+      "name": "apiKey",
+      "in": "query",
+      "description": "An alternative for authentication, but it's now deprecated. Please utilize Microsoft Entra ID as the preferred option.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "IsSubscribedOperationRequestParameters.configurationEtag": {
+      "name": "x-ms-qps-configuration-etag",
+      "in": "header",
+      "description": "An encoded string that indicates whether the collection configuration is changed.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "configurationEtag"
+    },
+    "IsSubscribedOperationRequestParameters.ikey": {
+      "name": "ikey",
+      "in": "query",
+      "description": "The ikey of the target Application Insights component for which the client checks whether there's any subscription to it.",
```

Comment:
Does "ikey" mean "instrumentation key"? If so, please spell that out in the description.